src/index.html

```diff
@@ -1083,7 +1083,7 @@ <h3 class="facility-info__title">Loading...</h3>
                   </svg></div>
               </div>
               <div class="point-data__h1_name">
-                <div class="truncate">Category Name</div>
+                <div class="truncate">Category Name asdas as dsa dsa das </div>
```

Comment:
This is a placeholder right, should it rather be named Category Name placeholder or something that isn't random letters?